community/front-end/ofe/infrastructure_files/workbench_tf/google/README.md

```diff
@@ -35,7 +35,7 @@ limitations under the License.
 | Name | Source | Version |
 |------|--------|---------|
 | <a name="module_project_radlab_ds_analytics"></a> [project\_radlab\_ds\_analytics](#module\_project\_radlab\_ds\_analytics) | terraform-google-modules/project-factory/google | ~> 11.0 |
-| <a name="module_vpc_ai_notebook"></a> [vpc\_ai\_notebook](#module\_vpc\_ai\_notebook) | terraform-google-modules/network/google | ~> 3.0 |
+| <a name="module_vpc_ai_notebook"></a> [vpc\_ai\_notebook](#module\_vpc\_ai\_notebook) | terraform-google-modules/network/google | ~> 7.0 |
```

Comment:
What is this update?

Reply:
This is one of the modules which explicitly specify a requirement on a lower version of the google provider.  Once I upgraded the provider version in our modules to the latest (v4.69.1), they conflicted with the requirements of the modules we reference, resulting in the following error message when running `terraform init`:

```
│ Error: Failed to query available provider packages
│ 
│ Could not retrieve the list of available versions for provider hashicorp/google: no available releases match the given
│ constraints >= 3.53.0, ~> 3.53, ~> 4.69.1, < 5.0.0
```

Upgrading the modules so that there's no conflict in the required version of the "hashicorp/google" provider allowed me to run `terraform init` without hitting the above error.